docs/sphinx/examples/python/ionq

```diff
@@ -0,0 +1,47 @@
+import cudaq
+
+# You only have to set the target once! No need to redefine it
+# for every execution call on your kernel.
+# To use different targets in the same file, you must update
+# it via another call to `cudaq.set_target()`
+cudaq.set_target("ionq")
+
+# Create the kernel we'd like to execute on IonQ
+kernel = cudaq.make_kernel()
+qubits = kernel.qalloc(2)
+kernel.h(qubits[0])
+kernel.cx(qubits[0], qubits[1])
+kernel.mz(qubits[0])
+kernel.mz(qubits[1])
+
+# Execute on IonQ and print out the results.
+
+# Option A:
+# By using the synchronous `cudaq.sample`, the execution of
+# any remaining classical code in the file will occur only
+# after the job has been returned from IonQ.
+counts = cudaq.sample(kernel)
```

Comment:
I also might suggest to move the synchronous call to the bottom, and maybe print something like "executing synchronously" right before, so that it doesn't look like the program just hangs for no reason when there is a significant queue.